### PR TITLE
Change hamburger

### PR DESCRIPTION
**Pull request by GantMan**
## Please verify the following:
- [x] Everything works on iOS/Android
- [x] `ignite-base` **ava** tests pass
- [x] `fireDrill.sh` passed

## Describe your PR
Add example of overriding and changing buttons

# Changes hamburger bar from this:
simulator screen shot jul 22 2016 3 08 13 pm

# To this
simulator screen shot may 30 2016 12 12 28 pm

### CODE REVIEW

**Comment by GantMan** [comment]
@skellock  and @kevinvangelder  - This structure make sense?

File: ignite-base/App/Navigation/NavItems.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  NavigationActions.refresh({
+    key: 'drawer',
+    open: value => !value
+  })
+}
+
+export default {
+  backButton () {
+    return (
+      <TouchableOpacity onPress={() => NavigationActions.pop}>
```
Comment (skellock) [inline ignite-base/App/Navigation/NavItems.js]: This can be just `{NavigationActions.pop}`.

**Comment by skellock** [comment]
👍

File: ignite-base/App/Navigation/NavItems.js (hunk 1 above)
Comment (GantMan) [inline ignite-base/App/Navigation/NavItems.js]: ah yes, good catch

**Pull request merged**